documentation/Home.asciidoc

```diff
@@ -12,9 +12,9 @@ Due to a template-based approach, CobiGen *generates any set of text-based docum
 **Input (currently):**
 
 * Java classes
-* XML-based files
+* `XML`-based files
```

Comment:
these are names like XML / WSDL / TypeScript / Ext JS. We should take them in our dictionary instead of making them look like code snippets. I think this was a miss understanding. Please correctly differentiate names from code like Classnames etc